app/assets/master.css

```diff
@@ -0,0 +1,73 @@
+html, body, p, th, td, li, dd, dt, pre {
+ font: 1em Verdana, Arial, Helvetica, sans-serif;
+ }
+
+div.login_bar {
+ z-index: 99999;
+ float:right; font-size:0.9em; border:1px;
+ line-height: 50px;	//size of the google+ user image
+ vertical-align:top;
+}
+
+.error { color:#FF0000; }
```

Comment:
> would it be a good idea to do that as a seperate PR

would prefer to do it early as css has a tendency to be left alone once its been written for fear of breaking things in super subtle ways... Your call.

I'd do the linting as part of the CI process. Personally I find pre-commit hooks annoying, for example when you're just hacking away and want to push a WIP branch up... There's also no guarantee they'll be added to everyone's version of the repo too. Again, your call.